database.js

```diff
@@ -96,6 +96,10 @@ module.exports = {
         const databaseURI = postgresCS || process.env.databaseuri;
 
         if (databaseURI) {
+            //logging is enabled by default
+            if(!logging) {
```

Comment:
This is absurd. If I set logging to true, this will execute, setting dbConfig.logging to false. I don't get this logic, at all. Also you just changed the signatures of very important functions just to pass this parameter? Why not use the same approach as everything else in life through env.vars in the config file? Seems like a very bad deal to me.

Opinions @kordianbruck @pfent